sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/EventProcessorClient.cs

```diff
@@ -1419,6 +1429,35 @@ private static EventProcessorOptions CreateOptions(EventProcessorClientOptions c
             };
         }
 
+        /// <summary>
+        /// Creates, starts, and enriches processing diagnostics scope.
+        /// </summary>
+        /// <param name="eventData">The instance of <see cref="EventData"/> which is being processed.</param>
+        /// <returns>The instance of scope.</returns>
```

Comment:
```suggestion
        /// <summary>
        ///    Creates, starts, and enriches a processing diagnostics scope.
        /// </summary>
        ///
        /// <param name="eventData">The instance of <see cref="EventData"/> which is being processed.</param>
        ///
        /// <returns>The instance of <see cref="DiagnosticScope"/>.</returns>
        ///
```

nit:  Jesse is annoying about white space in these because he can't read dense comments.